src/services/regionToJnvMapping.js

```diff
@@ -492,6 +492,7 @@ export const jnvState = {
     "JNV Aurangabad",
     "JNV Washim",
     "JNV Yavatmal",
+    "SEED",
```

Comment:
Maybe make it capitalize, instead of full caps.

Reply:
JNV team wanted it to be all-caps 🤷